chain/network/src/routing/routing_table_actor.rs

```diff
@@ -253,8 +253,8 @@ impl RoutingTableActor {
             self.peer_last_time_reachable.insert(peer.clone(), now);
         }
 
-        near_metrics::inc_counter_by(&metrics::ROUTING_TABLE_RECALCULATIONS, 1);
-        near_metrics::set_gauge(&metrics::PEER_REACHABLE, self.peer_forwarding.len() as i64);
```

Comment:
I guess we now should drop these helpers? That is, just remove `near_metrics::set_gauge`, `near_metrics::inc_counter_by` and the like?

Reply:
@matklad They are used in a lot of places. More PR's are needed.

Reply:
Yeah, that's what I mean: let's be consistent and panic on init for *every* gauge, and not just for a subset.